draft-ietf-quic-transport.md

```diff
@@ -2471,13 +2471,13 @@ considerations.  MAX_STREAM_ID frames constitute minimal overhead, while
 withholding MAX_STREAM_ID frames can prevent the peer from using the available
 parallelism.
 
-Implementations will likely want to increase the maximum stream ID as
-peer-initiated streams close.  Sending a MAX_STREAM_ID frame along with ACK
-frames ensures a regular increase in the stream limit without generating excess
+Sending any increase to the maximum stream ID in the same packet as an ACK frame
+ensures a regular increase in the stream limit without generating excess
```

Comment:
Yep, I think that I will just remove the paragraph.  The advice on when to send is still present in the next paragraph, which is what you are looking for.